doc/en/reference/reference.rst

```diff
@@ -1892,8 +1892,12 @@ All the command-line flags can be obtained by running ``pytest --help``::
                             tests. Optional argument: glob (default: '*').
       --cache-clear         Remove all cache contents at start of test run
       --lfnf={all,none}, --last-failed-no-failures={all,none}
-                            Which tests to run with no previously (known)
-                            failures
+                            Determines whether to execute tests when there
+                            are no previously (known) failures or when no
+                            cached ``lastfailed`` data was found.
+                            This option governs the behavior of ``--if``.
+                            Default ``all`` runs all tests with no known failures.
+                            ``none`` avoids tests and exits if no failures.
```

Comment:
Some minor tweaks:

```suggestion
                            With ``--lf``, determines whether to execute tests when there
                            are no previously (known) failures or when no
                            cached ``lastfailed`` data was found.
                            ``all`` (the default) runs the full test suite again.
                            ``none`` just emits a message about no known failures and exits successfully.
```

Reply:
Thanks for the feedback, update the code here: 75e815f6c1dd5705c3a1f6d11b320013775b59e4